Jellyfin.Server/SocketSharp/SharpWebSocket.cs

```diff
@@ -117,6 +111,7 @@ public Task SendAsync(string text, bool endOfMessage, CancellationToken cancella
         public void Dispose()
         {
             Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
can you explain why?

Reply:
`Objects that implement the IDisposable interface can call this method from the object's IDisposable.Dispose implementation to prevent the garbage collector from calling Object.Finalize on an object that does not require it. Typically, this is done to prevent the finalizer from releasing unmanaged resources that have already been freed by the IDisposable.Dispose implementation.`
https://docs.microsoft.com/en-us/dotnet/api/system.gc.suppressfinalize?redirectedfrom=MSDN&view=netframework-4.7.2#System_GC_SuppressFinalize_System_Object_

TLDR: Don't queue for finalization when derived classes implement a finalizer.